### t-test Hypothesis Testinng

Whenever we are comparing sample mean with Population mean we are performing t-test
Types of t-test : 
1. 1- sample t-test (from scipy.stats import ttest_1samp)
2. 2- sample t-test (from scipy.stats import ttest_2samp)

Formula : 
1. t = (x - mu)/(s/square-root of n)
    1. x - sample mean
    2. mu - population mean
    3. s - sample standard deviation
    4. n - sample size

In [1]:
from scipy.stats import ttest_1samp

In [2]:
sample_salary = [52000,51000,49500,53000,48000,50000,50500,51500,49800,49000] # sample salaries in company
avg_salary= 50000 # this is mean of population - avg salary of employees in company

Now a we have 1 sample and population mean salary apply t-test formula to calculate 

In [3]:
ttest_1samp(sample_salary, avg_salary)

TtestResult(statistic=np.float64(0.9135161792978956), pvalue=np.float64(0.3847901317073621), df=np.int64(9))

In response we get 
1. statistic=np.float64(0.9135161792978956), 
2. pvalue=np.float64(0.3847901317073621), 
3. df=np.int64(9)